rhizome/host/lib/vm_setup.rb

```diff
@@ -490,44 +491,8 @@ def download_boot_image(boot_image, custom_url: nil, ca_path: nil)
     }
 
     download = urls.fetch(boot_image) || custom_url
```

Comment:
What do you think about moving `urls` hash to the `BootImage` class too?

Reply:
I thought more about this & decided to keep URLs & sha256 sums in the rhizome side. So, I also did (a version of) what you suggested.